includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -28,15 +28,24 @@ class WC_REST_Payments_Orders_Controller extends WC_Payments_REST_Controller {
 	 */
 	private $gateway;
 
+	/**
+	 * WC_Payments_Customer instance for working with customer information
+	 *
+	 * @var WC_Payments_Customer_Service
+	 */
+	private $customer_service;
+
 	/**
 	 * WC_Payments_REST_Controller constructor.
 	 *
-	 * @param WC_Payments_API_Client   $api_client - WooCommerce Payments API client.
-	 * @param WC_Payment_Gateway_WCPay $gateway - WooCommerce Payments payment gateway.
+	 * @param WC_Payments_API_Client       $api_client - WooCommerce Payments API client.
+	 * @param WC_Payment_Gateway_WCPay     $gateway - WooCommerce Payments payment gateway.
+	 * @param WC_Payments_Customer_Service $customer_service - Customer class instance.
```

Comment:
I believe we follow this sort formatting in doc-blocks (I know it's not consistent though):

```
	 * @param WC_Payments_API_Client              $api_client             WooCommerce Payments API client.
	 * @param WC_Payment_Gateway_WCPay     $gateway               WooCommerce Payments payment gateway.
	 * @param WC_Payments_Customer_Service $customer_service Customer class instance.
```

Reply:
Fixed.